### PR TITLE
[GBI-1172] - Fix Healthcheck on E2E Test

### PR DESCRIPTION
**Pull request by manfioLP**
https://rsklabs.atlassian.net/browse/GBI-1172

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -9,6 +9,11 @@ jobs:
   e2e-test:
     runs-on: ubuntu-18.04
     steps:
+      - name: Set up Go 1.18
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```
Comment (Luisfc68) [inline .github/workflows/e2e.yml:15]: Maybe is better to use 1.20 version? since its the one specified in readme

Comment (manfioLP) [reply .github/workflows/e2e.yml:15]: you['re right! thanks luis :)

Comment (manfioLP) [reply .github/workflows/e2e.yml:15]: actually go-version 1.2 works a bit oddly in GH actions, I will leave it to 1.18 until future updates

**Review by rafaiovlabs — APPROVED** [review]
LGTM

**Pull request merged**